lib/layout.js

```diff
@@ -174,11 +172,21 @@ const pickZone = (opts, frameBounds, targetBounds, size) => {
     { side: `end`, standing: `below`, flow: `column`, order: 1, w: f.x2, h: (f.y2 - t.y2) },
     { side: `start`, standing: `left`, flow: `row`, order: -1, w: t.x, h: f.y2 },
   ]
-
+  
+  /* Order the zones by the amount of popup that would be cut out if that zone is used.
+     The first one in the array is the one that cuts the least amount.
+      
+     const area = size.w * size.h  // Popup area is constant and it does not change the order
+  */
+  zones.forEach((z) => {   
```

Comment:
This ordering is only used if there are no available zones. Therefore we should only compute it when there are no available zones.